engine/userguide/networking/work-with-networks.md

```diff
@@ -787,6 +787,12 @@ containers connected to that network can access the alias.
 Multiple containers can share the same network-scoped alias within the same
 network. This example illustrates how this works.
 
+> **Note**: Network aliases are a legacy networking feature. Swarm services
```

Comment:
Network aliases in itself are not legacy, they're perfectly ok to use (i.e. if my container is named "foobar", but I want to access it by different name(s).

For load-balancing, dnsrr may not always lead to the expected result (e.g. nginx will cache the result so may still always connect to the same container)